src/app-layout/index.tsx

```diff
@@ -689,7 +689,7 @@ const OldAppLayout = React.forwardRef(
                 topOffset={headerHeight}
                 bottomOffset={footerHeight}
                 displayed={splitPanelDisplayed}
-                width={splitPanelDisplayed ? splitPanelSize : undefined}
+                width={splitPanelOpen && splitPanel ? splitPanelSize : undefined}
```

Comment:
I wonder why is there a difference between `splitPanelDisplayed` and `splitPanelOpen && splitPanel`? It's a bit confusing :)